<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QWidget" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Albert settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1">
         <item>
          <widget class="QGroupBox" name="groupBox_hotkey">
           <property name="title">
            <string>Hotkey</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <layout class="QHBoxLayout" name="hl_hotkey" stretch="0,0,0">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="GrabKeyButton" name="grabKeyButton_hotkey">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Appearance</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="hl_onTop">
              <item>
               <widget class="QLabel" name="label_onTop">
                <property name="text">
                 <string>Always on &amp;top:</string>
                </property>
                <property name="buddy">
                 <cstring>checkBox_onTop</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_onTop">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hl_hideOnFocusOut">
              <item>
               <widget class="QLabel" name="label_hideOnFocusOut">
                <property name="text">
                 <string>Hide on &amp;focus out:</string>
                </property>
                <property name="buddy">
                 <cstring>checkBox_hideOnFocusOut</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_hideOnFocusOut">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hl_center">
              <item>
               <widget class="QLabel" name="label_center">
                <property name="text">
                 <string>Always &amp;center Albert:</string>
                </property>
                <property name="buddy">
                 <cstring>checkBox_center</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_center">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hl_scrollbar">
              <item>
               <widget class="QLabel" name="label_scrollbar">
                <property name="text">
                 <string>Show &amp;scrollbar:</string>
                </property>
                <property name="buddy">
                 <cstring>checkBox_scrollbar</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_scrollbar">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hl_icons">
              <item>
               <widget class="QLabel" name="label_icons">
                <property name="text">
                 <string>Show icons:</string>
                </property>
                <property name="buddy">
                 <cstring>checkBox_icons</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_icons">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hl_maxProp">
              <item>
               <widget class="QLabel" name="label_proposals">
                <property name="text">
                 <string>Number of &amp;results:</string>
                </property>
                <property name="buddy">
                 <cstring>spinBox_proposals</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_proposals">
                <property name="suffix">
                 <string/>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hl_style">
              <item>
               <widget class="QLabel" name="label_style">
                <property name="text">
                 <string>The&amp;me:</string>
                </property>
                <property name="buddy">
                 <cstring>comboBox_themes</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_themes"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resources.qrc">:/app_icon</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPlugins">
      <attribute name="title">
       <string>Plugins</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListView" name="listView_plugins">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget_pluginInfos" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_placeholder">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Select a plugin.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAbout">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="about_icon">
         <property name="pixmap">
          <pixmap resource="../resources.qrc">:/app_icon</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="about_text">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;font-size:12pt; font-weight:600;&quot;&gt;Albert omni launcher ___versionstring___&lt;/p&gt;
&lt;p&gt;If you encounter bugs or think you have suggestions for improvement, tell me at the &lt;/span&gt;&lt;a href=&quot;https://github.com/ManuelSchneid3r/albert/issues&quot;&gt;github issue tracker.&lt;/a&gt;&lt;br&gt; If you really like this software please consider a donation via &lt;a href=&quot;https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&amp;hosted_button_id=W74BQPKPGNSNC&quot;&gt;PayPal&lt;/a&gt; or &lt;a href=&quot;https://flattr.com/submit/auto?user_id=manuelschneid3r&amp;url=https%3A%2F%2Fgithub.com%2FManuelSchneid3r%2Falbert&quot;&gt;Flattr&lt;/a&gt;. &lt;/p&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GrabKeyButton</class>
   <extends>QPushButton</extends>
   <header>grabkeybutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
